src/styles/partials/components/_modal.scss

```diff
@@ -0,0 +1,76 @@
+.hidden {
+  display: none;
+}
+
+[role="alertdialog"],
+[role="dialog"] {
+  box-sizing: border-box;
+  padding: 15px;
+  color: $white;
+  background-color: $pale-blue;
+  min-height: 100vh;
+  padding: $padding-large $padding-jumbo;
+
+  * {
+    color: white;
+  }
+}
+
+@media screen and (min-width: 640px) {
+  [role="alertdialog"],
+  [role="dialog"] {
+    position: absolute;
+    top: 2rem;
+    left: 50vw; /* move to the middle of the screen (assumes relative parent is the body/viewport) */
+    transform: translateX(
+      -50%
+    ); /* move backwards 50% of this element's width */
+    min-width: calc(640px - (15px * 2)); /* == breakpoint - left+right margin */
+    min-height: auto;
+    box-shadow: 0 19px 38px rgba(0, 0, 0, 0.12), 0 15px 12px rgba(0, 0, 0, 0.22);
+  }
+}
+
+.dialog_close_button {
+  float: right;
+  position: absolute;
+  top: 10px;
+  left: 92%;
+  height: 25px;
+}
+
+.dialog_close_button img {
+  border: 0;
+}
+
+/* native <dialog> element uses the ::backdrop pseudo-element */
+
+/* dialog::backdrop, */
+.dialog-backdrop {
+  display: none;
+  position: fixed;
+  overflow-y: auto;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+@media screen and (min-width: 640px) {
+  .dialog-backdrop {
+    background: rgba(0, 0, 0, 0.3);
```

Comment:
this should go in a variable somewhere

Reply:
moved this to an overlay variable file, this will be used for the site navigation hamburger menu as well which made a good case for the variable.